examples/ethers/src/index.ts

```diff
@@ -0,0 +1,88 @@
+import SDK, { JRPC } from '@rpch/sdk';
+import { JsonRpcProvider, JsonRpcPayload, JsonRpcResult } from 'ethers';
+import dotenv from 'dotenv';
+dotenv.config();
+/**
+ * The RPCh ethers adapter is an extension of the original
+ *  JsonRpcProvider which allows clients to use drop-in and replace,
+ * so they can send their RPC requests through the RPCh network.
+ */
+export class RPChProvider extends JsonRpcProvider {
+    constructor(
+        public readonly url: string,
+        public readonly sdk: SDK,
+    ) {
+        super(url);
+    }
+
+    async send(method: string, params: Array<any> | Record<string, any>): Promise<any> {
```

Comment:
isn't that the default implementation?
We don't need to override this function then, right?

Reply:
this is correct, going to remove it.